README.md

```diff
@@ -1,2 +1,66 @@
-# mongoose-storage-driver-kafka
+# Introduction
+
 Kafka storage driver for Mongoose
+
+# Features
++ Item types:
+  * `data item` - Record (ProducerRecord, ConsumerRecord)
+  * `path` - Topic
++ Data item operation types:
+  * `create`
+  * `read`
++ Path item operation types:
+    * `create`
+    * `read`
+    * `delete`
+    * `list`
+`
+# Design
+
+| Kafka | Mongoose |
+|---------|----------|
+| Record | *Date Item* |
+| Topic | *Path Item* |
+| Partition | N/A |
+## Record Operations
+
+Mongoose should perform the load operations on the *records* when the configuration option `item-type` is set to `data`.
+
+### Create
+`ProducerApi` has a `KafkaProducer` class with function  [send()](http://kafka.apache.org/21/javadoc/org/apache/kafka/clients/producer/KafkaProducer.html#send-org.apache.kafka.clients.producer.ProducerRecord-), which can send a record to topic.
```

Comment:
Suggesting using the method
http://kafka.apache.org/21/javadoc/org/apache/kafka/clients/producer/KafkaProducer.html#send-org.apache.kafka.clients.producer.ProducerRecord-org.apache.kafka.clients.producer.Callback-
instead because we need a calback to know when the request is finished in order to measure the request latency.

Reply:
Fixed